knimin/tests/test_ag_edit_barcode.py

```diff
@@ -8,6 +8,10 @@
 
 
 class TestAGEditBarcodeHandler(TestHandlerBase):
+    # these fields switch None representation and thus are hard to check
+    # for equality
+    none_fields = ['environment_sampled', 'withdrawn', 'refunded']
```

Comment:
I haven't had too much time to investigate this, but I saw that different methods represent the same field differently. Sometimes it's 'N', or None or '' or 'None'.
Since it did not affect my tests here, I did not want to spend more time with it here.